Azure/communication-ui-library-android
# [Bug] Making text or graphical representation available for the mic on participants

## Purpose

* Making text or graphical representation available for the participants, currently communicating, with .MIC ON'.
* bug description: https://skype.visualstudio.com/SPOOL/_workitems/edit/2763088

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test


* Turn on the Talkback
* Navigate to call setup page
* Navigate to Join call and activate it
* Navigate to the floating header and activate participant list
* Navigate in the list with current call setting for each participant
* Observe the issue

## What to Check
Verify that the following are valid
* Participants'(currently communicating) with MIC ON visual or programmatical representation should be present.
